src/System.Security.Cryptography.Algorithms/tests/RandomNumberGeneratorTests.cs

```diff
@@ -151,7 +151,7 @@ private static void DifferentSequential(int arraySize)
             // = 1/256 * 1/256 * ... * 1/256
             // = 1/(256^10)
             // = 1/1,208,925,819,614,629,174,706,176
-            Assert.NotEqual(first, second);
+            Assert.NotEqual<byte>(first, second);
```

Comment:
You are right, this change doesn't have any measurable difference.  Assert.[Not]Equal checks for IEnumerable underneath the covers and does the enumeration.  This is different behavior than MSTest, which is why I was confused.
I'll revert these changes.